src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -65,16 +64,17 @@ private static String helpify(String help){
 
     static String describeType(ParameterType type, String prefix) throws Exception {
         StringBuilder typeInfo = new StringBuilder();
-        if (type instanceof AtomicType) {
```

Comment:
This doesn't seem to change the ordering, just no longer support atomic type. Is that intentional?

Reply:
The atomic type is moved to the getTypeDescription method which is called before all the help and further nesting is generated.